common.eai

```diff
@@ -6084,20 +6074,20 @@ function GetCreepGuardGroup takes integer nn returns nothing
   local integer i = 0
   local unit u = null
   local group g = null
-  if nearest_neutral[nn] == null then
-    return
```

Comment:
Removing return is less efficient as it now has to evaluate entire function instead of exiting out of it early.

Reply:
no remove,  just changed it to not empty before proceeding.   Let's look at the next two lines